runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -977,13 +977,12 @@ void BaseLoader<LoaderDomain>::loadBatchMatMul(const Operator *op, ir::Graph &su
   switch (builtin_op)
   {
     case BuiltinOperator::BuiltinOperator_BATCH_MATMUL:
-      // Different option name makes build fail
-      // Circle: adjoint_lhs, adjoint_rhs
-      // TFLite: adj_x, adj_y
-      // TODO Fix this issue
-      param.adj_x = op->builtin_options_as_BatchMatMulOptions()->adjoint_lhs();
-      param.adj_y = op->builtin_options_as_BatchMatMulOptions()->adjoint_rhs();
-      break;
+      // Handled on each loader: different option name
+      //  Circle: adjoint_lhs, adjoint_rhs
+      //  TFLite: adj_x, adj_y
```

Comment:
Wow, they changed option name field .....

Reply:
Maybe we used `adjoint_lhs, adjoint_rhs` from under development code and then TFlite released with `adj_x, adj_y`

Reply:
Right. We introduced them from `v2.2.0-rc2`, but option field names are changed on release version.